thorlcr/graph/thgraphmaster.cpp

```diff
@@ -2720,6 +2721,11 @@ void CMasterGraph::handleSlaveDone(unsigned node, MemoryBuffer &mb)
         sdMb.setBuffer(len, (void *)d);
         act->slaveDone(node, sdMb);
     }
+    offset_t activeSpillSize, nodeGraphSpill;
+    mb.read(nodeGraphSpill);
+    mb.read(activeSpillSize);
+    totalActiveSpillSize += activeSpillSize;
```

Comment:
Where do these get reset - e.g., if this is a spill within a LOOP will it incorrectly add up the spill file sizes for the all the  iterations.  @jakesmith not sure where that reset code would go, or 100% sure how spills in loops are handled.

Reply:
I think 'totalActiveSpillSize' needs resetting in CMasterGraph::preStart.
It will be recalculated each LOOP iteration, i.e. the current active total.

Would be a good to test that it does what's expected though in a LOOP that has spills within it.